### ISSUE TITLE
Uninstall not possible 

### ISSUE THREAD

**Issue by derkrasseleo** [studiorack/studiorack-app#7]
I tried uninstalling diopser via the GUI and the uninstallation button got deactivated. After waiting a while, I restarted the app and the button still said "uninstall".

I also tried uninstalling via cli:

```
PS C:\Users\leoch> studiorack plugin uninstall studiorack/diopser/diopser
⎋ C:\Users\leoch\AppData\Roaming\studiorack\config.json
⌕ \Program Files (x86)\Common Files\**\*.{component,aax,rta,lv2,sfz,tdm,vst,vst3}
node:fs:1392
  handleErrorFromBinding(ctx);
  ^

Error: EPERM: operation not permitted, scandir 'C:/Program Files (x86)/Common Files/LearnHtml/HttpWackue'
    at Object.readdirSync (node:fs:1392:3)
    at GlobSync._readdir (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:285:46)
    at GlobSync._readdirInGlobStar (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:264:20)
    at GlobSync._readdir (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:273:17)
    at GlobSync._processReaddir (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:134:22)
    at GlobSync._process (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:129:10)
    at GlobSync._processGlobStar (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:377:10)
    at GlobSync._process (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:127:10)
    at GlobSync._processGlobStar (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:380:10)
    at GlobSync._process (C:\Users\leoch\AppData\Roaming\npm\node_modules\@studiorack\cli\node_modules\glob\sync.js:127:10) {
  errno: -4048,
  syscall: 'scandir',
  code: 'EPERM',
  path: 'C:/Program Files (x86)/Common Files/LearnHtml/HttpWackue'
}
```

**Comment by derkrasseleo** [comment]
Same error for dexed

**Comment by kmturley** [comment]
Found the cause of this issue. On other machines audio plugins are a specific format (vst, vst3). However on Windows they are commonly .dll

When StudioRack scanned for plugin files, it assumed .dll were always audio plugins which is not the case. It would then error when the DLL file did not pass the Steinberg validation tool.

I've modified the scanning loop to skip/ignore plugins which don't validate:
https://github.com/studiorack/studiorack-core/commit/9842717ccf96624fd2bba7ed46ca884181dc7d2b
